Makefile

```diff
@@ -18,4 +18,4 @@ lint:
 
 
 test: lint
-	go test .
```

Comment:
we should add codecov to this.  Can be a separate PR / tracking issue.

Reply:
will do